Client/Phpredis/ClientCluster.php

```diff
@@ -0,0 +1,1532 @@
+
+        parent::__construct(null, $seeds);
+    }
+
+    /**
+     * Proxy function.
+     *
+     * @param string $name      A command name
+     * @param array  $arguments Lit of command arguments
+     *
+     * @throws \RuntimeException If no Redis instance is defined
+     *
+     * @return mixed
+     */
+    private function call($name, array $arguments = array())
+    {
+        $startTime = microtime(true);
+        $result = call_user_func_array("parent::$name", $arguments);
+        $duration = (microtime(true) - $startTime) * 1000;
+
+        if (null !== $this->logger) {
+            $this->logger->logCommand($this->getCommandString($name, $arguments), $duration, $this->alias, false);
+        }
+
+        return $result;
+    }
+
+    /**
+     * Returns a string representation of the given command including arguments.
+     *
+     * @param string $command   A command name
+     * @param array  $arguments List of command arguments
+     *
+     * @return string
+     */
+    private function getCommandString($command, array $arguments)
+    {
+        $list = array();
+        $this->flatten($arguments, $list);
+
+        return trim(strtoupper($command).' '.implode(' ', $list));
+    }
+
+    /**
+     * Flatten arguments to single dimension array.
+     *
+     * @param array $arguments An array of command arguments
+     * @param array $list      Holder of results
+     */
+    private function flatten($arguments, array &$list)
+    {
+        foreach ($arguments as $key => $item) {
+            if (!is_numeric($key)) {
+                $list[] = $key;
+            }
+
+            if (is_scalar($item)) {
+                $list[] = strval($item);
+            } elseif (null === $item) {
+                $list[] = '<null>';
+            } else {
+                $this->flatten($item, $list);
+            }
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function ping($key_or_address)
+    {
+        return $this->call('ping', func_get_args());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function get($key)
+    {
+        return $this->call('get', func_get_args());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function set($key, $value, $timeout = null)
+    {
+        return $this->call('set', func_get_args());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setex($key, $ttl, $value)
+    {
+        return $this->call('setex', func_get_args());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function psetex($key, $ttl, $value)
+    {
+        return $this->call('psetex', func_get_args());
+    }    
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setnx($key, $value)
+    {
+        return $this->call('setnx', func_get_args());
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function del($key, ...$other_keys)
+    {
+        return $this->call('del', func_get_args());
```

Comment:
Same problem as stated above, with more than 1 argument it will come `['key1', ['key2', 'key3']]`